agent/api/task.go

```diff
@@ -140,10 +140,14 @@ func (task *Task) initializeEmptyVolumes() {
 				continue
 			}
 			if _, ok := vol.(*EmptyHostVolume); ok {
-				if container.SteadyStateDependencies == nil {
-					container.SteadyStateDependencies = make([]string, 0)
+				if container.TransitionDependencySet.ContainerDependencies == nil {
+					container.TransitionDependencySet.ContainerDependencies = make([]ContainerDependency, 0)
 				}
-				container.SteadyStateDependencies = append(container.SteadyStateDependencies, emptyHostVolumeName)
+				container.TransitionDependencySet.ContainerDependencies = append(container.TransitionDependencySet.ContainerDependencies, ContainerDependency{
+					ContainerName:   emptyHostVolumeName,
+					SatisfiedStatus: ContainerRunning,
```

Comment:
Can this be `ContainerCreated` for the empty host volume container?

Reply:
IIRC, it need to be `ContainerRunning` because old versions of Docker (1.5.x era) don't actually create the volumes until `Start` is called on the container.

Reply:
ah, let's leave it as-is then.